app/controllers/books_controller.rb

```diff
@@ -1,11 +1,14 @@
 class BooksController < ApplicationController
+  # before_action :authenticate_user!
   def index
     @books = Book.all
     render json: Kaminari.paginate_array(@books).page(params[:page])
   end
 
   def show
     @book = Book.find(params[:id])
-    render json: @book
+    render json: @book, status: 200
+  rescue ActiveRecord::RecordNotFound
+    render json: { error: 'not-found' }.to_json, status: 404
```

Comment:
Good, remember you can use the keys :ok =>200, :not_found =>404. 
The message that you return inside the json can be a little more descriptive like "the book dont exists or couldnt be found" but don't worry you are going to use this when you arrived to the i18n card.

Reply:
Thanks, I did it